### DISCUSSION TITLE
Missing autodiscovery entities in homeassistant  after fw update

### DISCUSSION THREAD

**Discussion by drlauridsen** [UtilitechAS/amsreader-firmware#518] [Q&A] [answered]
Not sure whether its a bug or just my setup that somehow failed, but after updating amsreader to 2.2.12, and Homeassistant to latest version, I am now missing some entities, as seen in screenshot.
Dont have access to the local network for a while, so not easy to find the error..
Using mqtt and autodiscovery, which worked fine until the updates..
Any ideas what went wrong?
![Screenshot_20230407_011624_Home Assistant](https://user-images.githubusercontent.com/61352332/230511144-0d08acda-cd15-45c6-bb97-9e6256387f10.jpg)

**Comment by gskjold** [comment, marked as answer]
This was an error in 2.2.12 and should be fixed in 2.2.13

**Comment by drlauridsen** [comment]
And indeed it is.. thank you - your hard work to keep this brilliant code working and with new features, is much appreciated.

**Reply by gskjold** [reply]
Good to hear, thank you! :)
